### PR TITLE
Add v2 exporter for clickhousemetricswrite

### PR DESCRIPTION
**Pull request by srikanthccv**
Part of https://github.com/SigNoz/signoz/issues/5975

Fixes https://github.com/SigNoz/signoz/issues/6064

### CODE REVIEW

File: exporter/clickhousemetricsexporterv2/fingerprint.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+	}
+	return h
+}
+
+// hashAddByte adds a byte to a fnv64a hash value, returning the updated hash.
+func hashAddByte(h uint64, b byte) uint64 {
+	h ^= uint64(b)
+	h *= prime64
+	return h
+}
```
[the remark this answers is not in the document]
Comment (srikanthccv) [reply exporter/clickhousemetricsexporterv2/fingerprint.go:25]: We are not changing how we calculate fingerprints. This is our existing mechanism for calculating fingerprints taken from Prometheus long ago. We can any deterministic technique to calculate fingerprints, and should even consider using the functions provided by CH when we consider changing it https://clickhouse.com/docs/en/sql-reference/functions/hash-functions#xxhash32-xxhash64

**Review by nityanandagohain — APPROVED** [review]
don't have much context, approving this to unblock.

Also do note that usage is not present here so when switching to v2, you will need to add the changes for reporting use.

**Comment by srikanthccv** [comment]
@nityanandagohain, yes, it was a deliberate choice because there is no way to suppress the usage from one exporter. When we completely deprecate the old, then we will switch the usage.

File: exporter/clickhousemetricsexporterv2/fingerprint.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (shivanshuraj1333) [reply exporter/clickhousemetricsexporterv2/fingerprint.go:25]: Is there a ticket for the same? If not do you mind creating one?

**Pull request merged**